orchagent/dash/dashrouteorch.cpp

```diff
@@ -76,9 +76,22 @@ bool DashRouteOrch::addOutboundRouting(const string& key, OutboundRoutingBulkCon
         SWSS_LOG_INFO("Retry as route group %s not found", ctxt.route_group.c_str());
         return false;
     }
-    if (ctxt.metadata.has_vnet() && gVnetNameToId.find(ctxt.metadata.vnet()) == gVnetNameToId.end())
+
+    std::string routing_type_str = dash::route_type::RoutingType_Name(ctxt.metadata.routing_type());
+    if (ctxt.metadata.routing_type() == dash::route_type::RoutingType::ROUTING_TYPE_VNET &&
```

Comment:
What about the case for ROUTING_TYPE_PRIVATELINK? We will want to return false in that case too, right?

Reply:
ROUTING_TYPE_PRIVATELINK is not allowed here, caught by line 106 - 111:

```
        SWSS_LOG_WARN("Routing type %s for outbound routing entry %s not allowed", routing_type_str.c_str(), key.c_str());
        return false;
```

It is handled at vnet mapping case - `addOutboundCaToPa`